packages/node-client/lib/types.ts

```diff
@@ -212,14 +212,6 @@ export interface UpdateSyncFrequencyResponse {
     frequency: string;
 }
 
-export interface StandardNangoConfig {
```

Comment:
can you re-export StandardNangoConfig from `@nangohq/types` to avoid breaking change?
Same for NangoSyncConfig that can be deleted and re-exported

Reply:
True, updated